src/shogun/CMakeLists.txt

```diff
@@ -217,6 +217,22 @@ CHECK_FUNCTION_EXISTS(lgammal HAVE_LGAMMAL)
 
 ############################ external dependencies
 
+#Stan math
+include(external/StanMath)
+
+SHOGUN_INCLUDE_DIRS(SCOPE PUBLIC SYSTEM
+  $<BUILD_INTERFACE:${STAN_INCLUDE_DIR_STAN_MATH}>
+  $<INSTALL_INTERFACE:include/shogun/lib/external/Stan>
```

Comment:
can you please rebase this over latest develop and change this to `include/shogun/third_party/Stan`